src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
   the original value `Get` will be stored in `http.request.method_original`.
   ([#5471](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5471))
 
+* Fixed the name of spans that have `http.request.method` attribute set to `_OTHER`.
+  Now, the span name have `HTTP` prefix, previously it was `_OTHER`.
```

Comment:
```suggestion
  The span name will be set as `HTTP {http.route}` as per the [specification](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/http/http-spans.md#name).
```

Reply:
Need the changelog entry for HttpClients too.

Reply:
No behavioral changes in HTTP client. Only common method is used.